testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/jdk/Zulu20JdkProvider.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.launcher.env.jdk;
+
+import io.trino.testing.containers.TestContainers;
+
+public class Zulu20JdkProvider
```

Comment:
Is there a reason to test with Zulu given that we're converging on temurin in all the other places ?

Reply:
Historic reasons. I'm not aware that we are moving everything to Temurin